analyses/cellranger-analysis/README.md

```diff
@@ -0,0 +1,58 @@
+# Pipeline for running and summarizing Cell Ranger count for single or multiple libraries for sc-/sn-RNA-Seq Analysis in 10X Genomics data
+
+## Usage
+
+To run all the scripts in this module from the command line sequentially, use:
+
+```
+bsub < submit-multiple-jobs.sh
+```
+
+The `submit-multiple-jobs.sh` script is designed to be run as if it was called from this module directory even when called from outside of this directory. 
+   - Step 1: To run the `j1.sh` script to align single or multiple libraries in parallel, i.e., `run-cellranger-analysis`. 
+   - Step 2: To run `j2.sh` to summarize alignment results, i.e., `summarize-cellranger-analysis`. The latter script will be on hold and executed once all libraries are aligned and `j1.sh` is complete. This is been taken care of by `waiter.sh` script.
+
+Parameters according to the project and analysis strategy will need to be specified in the following scripts:
+- `j1.sh`: 
+   - `--file`: define path with `project_metadata` file. This is by default in the `./input/` dir at the module. The file needs to be in `*.txt` file format. If the file lives in another folder, this should be modified accordingly. In addition, there is a code line to convert `*.tsv` file to `*.txt` file, if needed. User can specify the file path in there for the conversion. This file can contain either one or multiple samples as long as the following minimum parameters are provided: `ID`, `SAMPLE`, and `FASTQ` columns.
```

Comment:
I think overall these instructions are pretty clear. However, I think they might be made a bit clearer by reordering some of the instructions and being slightly more explicit. 
E.g. "define path with `project_metadata` file. By default in the `./input/` dir within the `/sc-rna-seq-snap/analyses/cellranger-analysis` module. If the file lives in another folder, this path should be modified accordingly within the script. The file needs to be in `*.txt` file format." 
Then leave the remaining text for this parameter as is.

Reply:
The `README` file has been updated accordingly. See 412b9fbba2601739985dfdbfae9fd97a5257635f.